social_django/fields.py

```diff
@@ -23,8 +24,13 @@
 
 field_class = functools.partial(six.with_metaclass, field_metaclass)
 
+if getattr(settings, 'SOCIAL_DJANGO_POSTGRES_JSONFIELD', False):
```

Comment:
Setting name should be `SOCIAL_AUTH_`, you can use `social_core.utils.setting_name` to build it like `setting_name('POSTGRES_JSONFIELD')` and it will give you `SOCIAL_AUTH_POSTGRES_JSONFIELD`.